kolibri/plugins/learn/assets/src/views/LibraryPage/UnPinnedDevices.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+
+  <div
+    class="card-main-wrapper"
+    :style="cardStyle"
+  >
+    <div class="" style="width:250px;height:100px;">
+      <h2 class="device-name">
+        <span>
+          <KIcon icon="mediaLiteracyResource" />
+        </span>
+        <span>
+          <TextTruncator
+            :text="deviceName"
+            :maxHeight="52"
+          />
+        </span>
+      </h2>
+      <p class="channels">
+        {{ channels }} channel(s)
+      </p>
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  import responsiveWindowMixin from 'kolibri.coreVue.mixins.responsiveWindowMixin';
```

Comment:
There is a useKResponsiveWindow composable that should be used in place of this mixin. Please see https://github.com/learningequality/kolibri/blob/1b5898310a58ce6e788d1d48a109b02ff29a41bb/kolibri/plugins/learn/assets/src/views/LibraryPage/index.vue for a usage example